docs/rest_api.rst

```diff
@@ -200,23 +200,81 @@ To delete a link:
     DELETE /course_access_groups/api/v1/group-courses/2/
 
 
-Marking Courses as Public
+Setting Courses as Public
 -------------------------
+When the course access group feature is enabled, by default all courses are
+private and only accessible to learners with memberships to groups that
+has those courses.
 
-These endpoints allow mark courses as a public.
-This means the course is available to
-all site learners regardless of their Course Access Groups membership.
+However, some users could have no group membership so they won't have access
+to the private courses. To a course available to those learners and everyone
```

Comment:
Something is needed between "To" and "a" in "To a course available". And wouldn't "those learners" "and everyone else" be the same set?

Here's a suggestion: "To make a course available to those learners, you should make your course public"  I'm not a big fan of using "make" twice (to make X you need to make Y), but at least it fills in the missing word between "To" and "a" without rethinking how we want to say this